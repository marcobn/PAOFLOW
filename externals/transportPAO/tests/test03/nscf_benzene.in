Isolate benzene molecule

   &CONTROL
      calculation = 'nscf',
      restart_mode='from_scratch',
      verbosity='default',
      prefix='benzene',
      nstep=100,
      tstress = .false.,
      tprnfor = .true.,
      pseudo_dir = '../pseudo/',
      outdir = './SCRATCH/'
      wf_collect = .TRUE.
   /
   &SYSTEM
      ibrav=8,
      celldm(1)=30.0000,
      celldm(2)=1.000000,    ! = 20.0 Bohr
      celldm(3)=1.000000,
      nat=12,
      ntyp=2,
      ecutwfc=45.0,
      occupations='smearing',
      smearing='gaussian',
      degauss=0.01,
      nspin=1,
      nbnd = 60
   /
   &ELECTRONS
      electron_maxstep = 100,
      mixing_mode = 'plain',
      mixing_beta = 0.3,
      conv_thr =  1.0d-8,
   /
ATOMIC_SPECIES
  C    12.011    C.blyp-mt.UPF
  H     1.008    H.blyp-vbc.UPF
ATOMIC_POSITIONS {angstrom}
H      -3.362   1.313   1.658
C      -1.581   2.525   1.645
H      -2.121   3.460   1.653
C      -0.181   2.524   1.633
H       0.359   3.459   1.631
C       0.518   1.311   1.622
H       1.598   1.311   1.612
C      -0.182   0.099   1.625
H       0.357  -0.836   1.616
C      -1.582   0.100   1.637
H      -2.123  -0.835   1.639
C      -2.282   1.313   1.648
K_POINTS {automatic}
   1 1 1    0  0  0

