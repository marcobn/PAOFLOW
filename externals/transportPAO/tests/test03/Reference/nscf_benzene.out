
     Program PWSCF v.5.0.2 (svn rev. 9392) starts on 17Apr2014 at 20:34: 7 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/nbgrp/npool/nimage =       6

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Reading input from /home/ferretti/want/tests/test16/nscf_benzene.in

     Atomic positions and unit cell read from directory:
     ./SCRATCH/benzene.save/
 

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        2148    2148    538               183436   183436   22924
     Max        2149    2149    540               183439   183439   22925
     Sum       12893   12893   3233              1100629  1100629  137545
 


     bravais-lattice index     =            8
     lattice parameter (alat)  =      30.0000  a.u.
     unit-cell volume          =   27000.0000 (a.u.)^3
     number of atoms/cell      =           12
     number of atomic types    =            2
     number of electrons       =        30.00
     number of Kohn-Sham states=           60
     kinetic-energy cutoff     =      45.0000  Ry
     charge density cutoff     =     180.0000  Ry
     Exchange-correlation      =  SLA  LYP  B88  BLYP ( 1 3 1 3 0)
     EXX-fraction              =        0.00

     celldm(1)=  30.000000  celldm(2)=   1.000000  celldm(3)=   1.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.000000 )  


     PseudoPot. # 1 for C  read from file:
     ../pseudo/C.blyp-mt.UPF
     MD5 check sum: 9707c83c4328ed89710f741247262d6d
     Pseudo is Norm-conserving, Zval =  4.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  279 points,  1 beta functions with: 
                l(1) =   0

     PseudoPot. # 2 for H  read from file:
     ../pseudo/H.blyp-vbc.UPF
     MD5 check sum: 64d25ad537c4b939b17faee1147b797b
     Pseudo is Norm-conserving, Zval =  1.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  243 points,  0 beta functions with: 

     atomic species   valence    mass     pseudopotential
        C              4.00    12.01100     C ( 1.00)
        H              1.00     1.00800     H ( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           H   tau(   1) = (  -0.2117753   0.0827070   0.1044389  )
         2           C   tau(   2) = (  -0.0995886   0.1590519   0.1036200  )
         3           H   tau(   3) = (  -0.1336036   0.2179484   0.1041239  )
         4           C   tau(   4) = (  -0.0114013   0.1589890   0.1028641  )
         5           H   tau(   5) = (   0.0226137   0.2178854   0.1027381  )
         6           C   tau(   6) = (   0.0326293   0.0825810   0.1021712  )
         7           H   tau(   7) = (   0.1006594   0.0825810   0.1015413  )
         8           C   tau(   8) = (  -0.0114643   0.0062361   0.1023602  )
         9           H   tau(   9) = (   0.0224877  -0.0526604   0.1017932  )
        10           C   tau(  10) = (  -0.0996516   0.0062991   0.1031161  )
        11           H   tau(  11) = (  -0.1337296  -0.0525974   0.1032420  )
        12           C   tau(  12) = (  -0.1437452   0.0827070   0.1038090  )

     number of k points=     1  gaussian smearing, width (Ry)=  0.0100
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000

     Dense  grid:  1100629 G-vectors     FFT dimensions: ( 135, 135, 135)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions        20.99 Mb     (  22925,   60)
        NL pseudopotentials             2.10 Mb     (  22925,    6)
        Each V/rho on FFT grid          6.40 Mb     ( 419175)
        Each G-vector array             1.40 Mb     ( 183439)
        G-vector shells                 0.03 Mb     (   3413)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions        83.95 Mb     (  22925,  240)
        Each subspace H/S matrix        0.88 Mb     ( 240, 240)
        Each <psi_i|beta_j> matrix      0.01 Mb     (      6,   60)

     The potential is recalculated from file :
     ./SCRATCH/benzene.save/charge-density.dat

     Starting wfc are   30 randomized atomic wfcs

     total cpu time spent up to now is        1.8 secs

     per-process dynamical memory:   120.2 Mb

     Band Structure Calculation
     Davidson diagonalization with overlap

     ethr =  3.33E-11,  avg # of iterations =100.0

     total cpu time spent up to now is      178.1 secs

     End of band structure calculation

          k = 0.0000 0.0000 0.0000     band energies (ev):

   -20.8026 -18.0881 -18.0842 -14.5145 -14.5139 -12.5671 -10.9957 -10.5465
    -9.9688  -9.9640  -8.5937  -7.9994  -7.9989  -5.8912  -5.8885  -0.8428
    -0.8411  -0.2976   0.3069   0.3074   0.3644   0.5462   0.5499   0.6375
     0.8970   0.9599   0.9617   1.1173   1.1190   1.1272   1.1642   1.1649
     1.1790   1.2383   1.2526   1.2643   1.6247   1.6354   1.7154   1.7265
     1.7587   1.7629   1.8107   1.8276   1.8589   1.8685   2.1462   2.2182
     2.2513   2.4571   2.5409   2.6319   2.6482   2.6507   2.6950   2.6982
     2.7549   2.7731   2.8057   2.8429

     the Fermi energy is    -2.9324 ev

     Writing output data file benzene.save
 
     init_run     :      1.59s CPU      1.68s WALL (       1 calls)
     electrons    :    175.12s CPU    176.33s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
     potinit      :      0.80s CPU      0.85s WALL (       1 calls)

     Called by electrons:
     c_bands      :    175.12s CPU    176.28s WALL (       1 calls)
     v_of_rho     :      0.56s CPU      0.59s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.01s CPU      0.01s WALL (       1 calls)
     cegterg      :    171.31s CPU    172.37s WALL (       5 calls)

     Called by *egterg:
     h_psi        :    129.65s CPU    130.19s WALL (     106 calls)
     g_psi        :      1.28s CPU      1.25s WALL (     100 calls)
     cdiaghg      :      2.39s CPU      2.35s WALL (     101 calls)

     Called by h_psi:
     add_vuspsi   :      0.64s CPU      0.65s WALL (     106 calls)

     General routines
     calbec       :      0.71s CPU      0.69s WALL (     106 calls)
     fft          :      0.49s CPU      0.50s WALL (      10 calls)
     fftw         :    113.10s CPU    113.76s WALL (    4508 calls)
     davcio       :      0.00s CPU      0.10s WALL (       3 calls)
 
     Parallel routines
     fft_scatter  :     32.13s CPU     32.45s WALL (    4518 calls)
 
     PWSCF        :  2m57.93s CPU     2m59.46s WALL

 
   This run was terminated on:  20:37: 7  17Apr2014            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
